package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ebury/chameleon-components",
-  "version": "1.9.4",
+  "version": "1.9.6",
```

Comment:
This change is correct. Npm had version 1.9.5 whilst here we had to 1.9.4. I had to jump 2 version up to catch up with npm